Kernel/Thread.h

```diff
@@ -1053,7 +1053,7 @@ class Thread
     static IterationDecision for_each(Callback);
 
     static constexpr u32 default_kernel_stack_size = 65536;
-    static constexpr u32 default_userspace_stack_size = 1 * MiB;
+    static constexpr u32 default_userspace_stack_size = 8 * MiB;
```

Comment:
A similar setting seems to be `stack_size` in `Kernel/API/Syscall.h`, which should probably be updated in lockstep. The `// Default PTHREAD_STACK_MIN` there is probably outdated though.

(Also, +1 for this change, as a default stack size of 4 MiB or more is required for self-hosting Serenity.)

Reply:
Instead of doing this, we now handle `PT_GNU_STACK` so that the executable can set its own stack size.